extractors/factory.go

```diff
@@ -0,0 +1,26 @@
+package extractors
+
+type FactoryFn func() Extractor
```

Comment:
FactoryFn is an abbreviation of Factory function right??

Reply:
yes 👍